### PR TITLE
K8SPSMDB-282 document PMM support

### PR DESCRIPTION
**Pull request by fiowro**
[K8SPSMDB-282](https://jira.percona.com/browse/K8SPSMDB-282)

### CODE REVIEW

File: source/monitoring.rst [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+to monitor Percona Server for MongoDB.
+
+.. note:: Only PMM 2.x versions are supported by the Operator.
+
+PMM is a client/server application. *PMM Client* runs on each node with the
+database you wish to monitor: it collects needed metrics and sends gathered data
+to *PMM Server*. As a user, you connect to PMM Server to see database metrics on
+a number of dashboards.
+
+That's why PMM Server and PMM Client are to be installed separately.
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:17]: /are to be/need to be/

File: source/monitoring.rst [hunk 2]
```diff
@@ -0,0 +1,107 @@
+.. _operator.monitoring:
+
+Monitoring
+==========
+
+The Percona Monitoring and Management (PMM) `provides an excellent
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:6]: /^The //

File: source/monitoring.rst [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+solution <https://www.percona.com/doc/percona-monitoring-and-management/2.x/index.html>`_
+to monitor Percona Server for MongoDB.
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:8]: /to monitor/of monitoring/

File: source/monitoring.rst [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+.. note:: Only PMM 2.x versions are supported by the Operator.
+
+PMM is a client/server application. *PMM Client* runs on each node with the
+database you wish to monitor: it collects needed metrics and sends gathered data
+to *PMM Server*. As a user, you connect to PMM Server to see database metrics on
+a number of dashboards.
+
+That's why PMM Server and PMM Client are to be installed separately.
+
+Installing the PMM Server
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:19]: /the//

File: source/monitoring.rst [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+-------------------------
+
+PMM Server runs as a *Docker image*, a *virtual appliance*, or on an *AWS instance*.
+Please refer to the `official PMM documentation <https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/index.html>`_
+for the installation instructions.
+
+Installing the PMM Client
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:26]: /the//

File: source/monitoring.rst [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+Kubernetes-based environment:
+
+#. The PMM client installation is initiated by updating the ``pmm``
+   section in the
+   `deploy/cr.yaml <https://github.com/percona/percona-server-mongodb-operator/blob/master/deploy/cr.yaml>`_
+   file.
+
+   -  set ``pmm.enabled=true``
+   -  ensure the ``serverHost`` (the PMM service name is
+      ``monitoring-service`` by default) is the same as value specified
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:39]: /as value/as the value/

File: source/monitoring.rst [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+      for the ``name`` parameter on the previous step, but with an added
+      additional ``-service`` suffix.
+   -  check that ``PMM_SERVER_USER`` key in the
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:42]: /that/that the/

File: source/monitoring.rst [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+      `deploy/secrets.yaml <https://github.com/percona/percona-server-mongodb-operator/blob/master/deploy/secrets.yaml>`_
+      secrets file contains your PMM Server user name (``admin`` by default).
+   -  make sure the ``PMM_SERVER_PASSWORD`` key in the
+      `deploy/secrets.yaml <https://github.com/percona/percona-server-mongodb-operator/blob/master/deploy/secrets.yaml>`_
+      secrets file contains password specified for the PMM Server during its
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:47]: /contains password/contains the password/

File: source/monitoring.rst [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+      installation
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:48]: Add a full stop

File: source/monitoring.rst [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+      
+      .. note:: You use ``deploy/secrets.yaml`` file to *create* Secrets Object.
+         The file contains all values for each key/value pair in convenient
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:51]: /in/in a/

File: source/monitoring.rst [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+   Apply changes with the ``kubectl apply -f deploy/secrets.yaml`` command.
+
+   When done, apply the edited ``deploy/cr.yaml`` file:
+
+   .. code:: bash
+
+      $ kubectl apply -f deploy/cr.yaml
+
+#. Check that correspondent Pods are
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:76]: /correspondent/corresponding/

File: source/monitoring.rst [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+   login/password authentication, and the browser is configured to `show
+   Percona Server for MongoDB
+   metrics <https://www.percona.com/doc/percona-monitoring-and-management/index.metrics-monitor.dashboard.html#pmm-dashboard-mongodb-list>`__.
+
+
+
+
+As you can see, because we use the ``stringData`` type when creating the Secrets
+object, all values for each key/value pair are stated in plain text format
+convenient from the user's point of view. But the resulting Secrets
```
Comment (PaulJacobs-percona) [inline source/monitoring.rst:99]: /convenient/which is convenient/

**Review by PaulJacobs-percona — COMMENTED** [review]
Various minor grammatical suggestions

File: source/monitoring.rst [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+#. The PMM client installation is initiated by updating the ``pmm``
+   section in the
+   `deploy/cr.yaml <https://github.com/percona/percona-server-mongodb-operator/blob/master/deploy/cr.yaml>`_
+   file.
+
+   -  set ``pmm.enabled=true``
+   -  ensure the ``serverHost`` (the PMM service name is
+      ``monitoring-service`` by default) is the same as the value specified
+      for the ``name`` parameter on the previous step, but with an added
+      additional ``-service`` suffix.
```
Comment (delgod) [inline source/monitoring.rst:41]: it was actual if PMM installed inside kubernetes, now it is just "hostname" of  PMM Server

**Review by delgod — CHANGES_REQUESTED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**